src/server/bin/www

```diff
@@ -4,37 +4,37 @@
  * Module dependencies.
  */
 
-var app = require('../app');
-var debug = require('debug')('_example:server');
-var http = require('http');
+const app = require("../app");
+const debug = require("debug")("_example:server");
+const http = require("http");
 
 /**
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
+const port = normalizePort(process.env.PORT || "3000");
+app.set("port", port);
 
 /**
  * Create HTTP server.
  */
 
-var server = http.createServer(app);
+const server = http.createServer(app);
 
 /**
  * Listen on provided port, on all network interfaces.
  */
 
 server.listen(port);
-server.on('error', onError);
-server.on('listening', onListening);
+server.on("error", onError);
+server.on("listening", onListening);
```

Comment:
Isn't `onListening` in the TDZ here? I think that'll throw

Reply:
TDZ?

Reply:
Temporal Dead Zone. `const` isn't hoisted like `var` is.

Reply:
Yup you were totally right, guess I'm used to webpack hoisting stuff 👍 

Fixed.